Cabal/Distribution/PackageDescription/Parsec.hs

```diff
@@ -363,6 +364,10 @@ parseName pos args = case args of
          parseFailure pos $ "Invalid name " ++ show args
          pure ""
 
+parseUnqualComponentName :: Position -> [SectionArg Position] -> ParseResult UnqualComponentName
+parseUnqualComponentName pos args = mkUnqualComponentName <$> parseName pos args
```

Comment:
You mean not expose it? It is used a couple time in that module if I am reading this right and it wasn't changed since this was merged.